src/components/binding/binding-panel.js

```diff
@@ -160,10 +160,10 @@ genotet.BindingPanel.prototype.initChrs_ = function() {
 
 /**
  * Adds a select2 for a new track.
+ * @param {number} trackIndex Index of the added track.
  * @private
  */
-genotet.BindingPanel.prototype.addTrack_ = function() {
-  var trackIndex = this.data.tracks.length - 1;
+genotet.BindingPanel.prototype.addTrack_ = function(trackIndex) {
```

Comment:
Ok. The other trackIndex's in this function are not in changes list so I didn't see them. Now I take a closer look and it seems problematic to me now.

Starting from line 168 we have
```js
var uiTrack = ui.find('#track-template').clone()
    .appendTo(ui)
    .attr('id', 'track-' + trackIndex)
    .show();
```
This is creating a new tract select at the end of the track select list. But right now your are taking specific trackIndex, which could be in the middle of the list. Then the UI selects order is wrong.

Have you ever tried to create multiple tracks of different binding data and check if they correspond correctly to the UI selects?